app/jobs/scraper/oregon_job.rb

```diff
@@ -0,0 +1,48 @@
+class Scraper::OregonJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    base = "https://www.oregon.gov"
+    apprenticeship_url = base + "/boli/apprenticeship/Pages/"
+    browser = Watir::Browser.new
+    browser.goto(apprenticeship_url + "apprenticeship-opportunities.aspx")
+    js_doc = browser.element(css: "tbody").wait_until(&:present?)
+    body = Nokogiri::HTML(js_doc.inner_html)
```

Comment:
@jenlu33 @FerPerales I think the spec is making the external URL call as far as I can tell? I wonder if the browser allows it to bypass the webmock configuration. I finally had to kill rspec in GH Actions after 4 hours as I think we will need a headless version to work there. Also, the spec alone took 5 minutes locally to run which is too long. 

Could we try stubbing everything? Something like this:

```
js_doc = file_fixture("scraper/some-final-html-after-the-js-has-run.html").read
element = double("element, wait_until: js_doc)
browser_mock = instance_double("Watir::Browser", goto: nil, element: element)
allow(Watir::Browser).to receive(:new).and_return(browser_mock)
```

And then when you copy the HTML file from the website after the JS has run, just remove all but a couple of the table rows so we only have to test that a couple of records are created.